src/Utilities/Compiler/Extensions/ISymbolExtensions.cs

```diff
@@ -466,6 +466,31 @@ public static bool IsImplementationOfAnyImplicitInterfaceMember<TSymbol>(this IS
             return false;
         }
 
+        /// <summary>
+        /// Checks if a given symbol implements an interface member implicitly
+        /// </summary>
+        public static bool IsImplementationOfAnyImplicitInterfaceMember<TSymbol>(this ISymbol symbol, out TSymbol interfaceMember)
```

Comment:
Is this a method move or a new method? Note that the implementation here is pretty expensive as we are iterating all interface methods in the type hierarchy and invoking `IsImplementationOfInterfaceMember` which does expensive work. If this extension method is called from a hot path, you certainly want to do performance measurements/profiling for this analyzer.

Reply:
Tagging @sharwell as he may have insights on the performance characteristics of this code.

Reply:
Yup. Called it out here: https://github.com/dotnet/roslyn-analyzers/pull/5155#discussion_r654071422

> you certainly want to do performance measurements/profiling for this analyzer

👍